system/puppet.py

```diff
@@ -147,6 +156,7 @@ def main():
             facter_basename=dict(default='ansible'),
             environment=dict(required=False, default=None),
             certname=dict(required=False, default=None),
+            tags=dict(required=False, default=None),
```

Comment:
Looks pretty, but I'm concerned about collision with ansible tags [2] in case of using list type.

[2] http://docs.ansible.com/ansible/playbooks_tags.html